DPG/Tasks/AOTTrack/V0Cascades/perfK0sResolution.cxx

```diff
@@ -346,14 +403,54 @@ struct perfK0sResolution {
   }
   PROCESS_SWITCH(perfK0sResolution, processData, "Process data", true);
 
-  void processMC(soa::Filtered<SelectedCollisions>::iterator const& collision, soa::Filtered<soa::Join<aod::V0Datas, aod::McV0Labels>> const& fullV0s,
-                 soa::Join<PIDTracks, aod::McTrackLabels> const& tracks, aod::McParticles const&)
+  // Running variables
+  o2::dataformats::VertexBase mVtx;
+  o2::dataformats::DCA mDcaInfoCovPos;
+  o2::dataformats::DCA mDcaInfoCovNeg;
+  o2::track::TrackParametrizationWithError<float> mTrackParCovPos;
+  o2::track::TrackParametrizationWithError<float> mTrackParCovNeg;
+
+  template <class TV0TracksTo, typename TV0>
+  void tuneV0(TV0 const& v0,
+              aod::McParticles const&,
+              aod::BCsWithTimestamps const& bcs)
+  {
+    initCCDB(bcs.begin());
+    trackTunedTracks->Fill(1, 2); // tune 2 tracks
+
+    const auto& posTrack = v0.template posTrack_as<TV0TracksTo>();
+    const auto& negTrack = v0.template negTrack_as<TV0TracksTo>();
+    setTrackParCov(posTrack, mTrackParCovPos);
```

Comment:
instead of colling this getters again, you can probably use those that you already call at the beginning of the `processMC (current lines 452, 453)